nixos/modules/hardware/gkraken.nix

```diff
@@ -3,16 +3,17 @@
 with lib;
 
 let
+  inherit (lib)
+    mdDoc mkEnableOption mkIf;
+
   cfg = config.hardware.gkraken;
 in
 {
   options.hardware.gkraken = {
-    enable = mkEnableOption (lib.mdDoc "gkraken's udev rules for NZXT AIO liquid coolers");
+    enable = mkEnableOption (mdDoc "gkraken's udev rules for NZXT AIO liquid coolers");
   };
 
   config = mkIf cfg.enable {
-    services.udev.packages = with pkgs; [
-      gkraken
-    ];
+    services.udev.packages = [ pkgs.gkraken ];
```

Comment:
For me the line before the change was so easy to understand that I can parse it at first glimpse without thinking. I can no longer do that with ``pkgs.`` especially if there are multiple.